scripts/theme-css.js

```diff
@@ -0,0 +1,22 @@
+const fs = require('fs-extra');
+const path = require('path');
+
+const supportedThemes = [
+    'sap_fiori_3',
+    'sap_fiori_3_dark',
+    'sap_fiori_3_hcb',
+    'sap_fiori_3_hcw',
+    'sap_fiori_3_light_dark'
+];
```

Comment:
I see this array used in two places.  Should we make a constants file that both can import?

Reply:
Yeah that's the plan! This is the _very_ rough version